packages/gitbook/src/components/RootLayout/CustomizationRootLayout.tsx

```diff
@@ -38,6 +45,7 @@ export async function CustomizationRootLayout(props: {
     const headerTheme = generateHeaderTheme(customization);
     const language = getSpaceLanguage(customization);
     const tintColor = getTintColor(customization);
+    const mixColor = getMixColor(customization.styling.primaryColor, tintColor);
```

Comment:
this is only about tint right?
```suggestion
    const mixColor = getTintMixColor(customization.styling.primaryColor, tintColor);
```